### ISSUE TITLE
populate the "go to line" dialog with the current line number

### ISSUE THREAD

**Issue by isidorn** [Microsoft/vscode#65219]
As an accessibility workaround for #52429
Also is useful for people who turn off line number like me.

**Comment by zersiax** [comment]
Awesome to see this, can't wait to see this in action :)

**Comment by isidorn** [comment]
@zersiax glad to hear!
@cleidigh might you be interested in working on this? Just checking...
I plan to implement it otherwise.

**Comment by cleidigh** [comment]
@isidorn 
Absolutely.  I need some work to do since SelectBox has absolutely ZERO ISSUES :-p HA HA !

I will start looking into it to get familiar with that code.

**Comment by cleidigh** [comment]
@isidorn 
@zersiax 
@alexandrudima 

Looked at the code and the comments from 

Questions/opinions:

- Assuming we add something like "Current Line: 148." to call out the current line, could be prefix or suffix:
   "Current Line: y.  Type a line number between 1 and {0} to navigate to x"
     "Type a line number between 1 and {0} to navigate to x.  Current Line: y"

- Should line numbers only be read if line numbers are turned on? @zersiax describes his use
  specifically what know why numbers are not displayed so I don't see how that would be good.

FWIW, my opinion:
- Put "Current Line: y'  at the beginning as I think it flows better.
- Add equivalent text for visuals ONLY IF line number display is off, otherwise speak only

Votes?

**Comment by alexandrudima** [comment]
I like your suggestion to always put the text "Current Line: y" at the beginning. I suggest we start with that and collect wider feedback (wait for bugs :) ).

**Comment by isidorn** [comment]
@cleidigh Yes, let start very simple with the Current Line prefix. Feel free to create a PR and I can review.
If the NVDA does not read it out, than that is an unrelated issue and @chrmarti would know more

**Comment by zersiax** [comment]
I could be horribly mistaken but I really don't think you need to do that?
If you just make sure the number is in the edit field to start with, highlighted so when you type it gets overridden pretty much like any other edit field, I think NVDA will just read it when the dialog comes up as it being the contents of the edit field. It's not really a stretch to figure out that it's talking about the current line, I would think? At that point adding "current line" is just extra verbosity.
Just my thoughts though :)

**Comment by isidorn** [comment]
Done by @cleidigh 
Thanks a lot! Closing
